persistence/db.go

```diff
@@ -43,6 +43,22 @@ type PostgresContext struct {
 	tx         pgx.Tx
 	blockstore kvstore.KVStore
 	txIndexer  indexer.TxIndexer
+	// DISCUSS: this might be retrieved from the block store - temporarily we will access it directly from the module
```

Comment:
I think this is more than fine for now. Let's merge it in because I want to get "a feel" for how we can do this well after I rebase.

```suggestion
	// DISCUSS(#284): this might be retrieved from the block store - temporarily we will access it directly from the module
```